frontend/src/app/components/UpdateList.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import classnames from 'classnames';
+import moment from 'moment';
+import LayoutWrapper from './LayoutWrapper';
+import { newsUpdateL10nId } from '../lib/utils';
+
+export function prettyDate(date) {
+  return moment(date).format('MMMM Do, YYYY');
```

Comment:
Based on [moment l18n docs](https://momentjs.com/docs/#/i18n/) It looked like all we needed to do was set `moment.locale(String);` which I did [here](https://github.com/mozilla/testpilot/pull/2495/files/c1f139a2346b08b3b839520f7523cf4f0acee903#diff-9e31d7481520a2040f72bb71e10130f1R23), though we'll need to test it out to be sure